lib/core/src/Cardano/Wallet.hs

```diff
@@ -758,7 +761,9 @@ newWalletLayer tracer bp db nw tl = do
                     ErrStartTimeLaterThanEndTime start end
             _ -> pure ()
         let tip = currentTip w ^. #slotId
-        liftIO $ assemble tip <$> DB.readTxHistory db (PrimaryKey wid)
+        liftIO $ assemble tip
+            <$> DB.readTxHistory db (PrimaryKey wid)
+                defaultTxSortOrder noFilter
```

Comment:
TODO: `_listTransactions` should take sort and filter parameters as well. 

`DB.readTxHistory` is well-tested through the `StateMachine` tests. Not sure we need to do anything in particular on the WalletLayer level 🤔